src/components/registerButton.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./../styles/registerButton.scss";
+
+const RegisterButton = ({to}) => (
+  <a target="_blank" rel="noopener noreferrer" href={to} className="register-button">Registrate aquí</a>
```

Comment:
No creo que sea necesario tener una propiedad `to`, teniendo en cuenta que en ambos casos va a apuntar a la misma URL y no va a tener otro uso diferente, se puede mockear la URL acá y ya.